tests/integration/api_test.py

```diff
@@ -346,6 +350,106 @@ def test_interface_with_last_used_should_be_listable(
     assert response.status_code == 200
 
 
+class TestVendorLookupGet:
```

Comment:
The lines 
```
endpoint = 'vendor'
create_token_endpoint(token, endpoint)
```
feel like something we can set up here for the whole class instead of repeating it in every test

Reply:
Changed it so it now uses a fixture